### ISSUE TITLE
Add support for XML response from stored procedures

### ISSUE THREAD

**Issue by jasneo** [ReversePOCO/EntityFramework-Reverse-POCO-Code-First-Generator#286]
The generated class variable is invalid. It will be good if we can just use the standard field name for XML response, perhaps label it as `XmlResponse`.

**Generated code**
```c#
    public class SpGetAllPersonsReturnModel
    {
        public System.String XML_F52E2B61-18A1-11d1-B105-00805F49916B { get; set; }
    }
```

**SQL script to create tables and stored procedure for error reproduction**
```sql
CREATE TABLE [dbo].[City](
	[CityID] [int] IDENTITY(1,1) NOT NULL,
	[Name] [varchar](50) NULL,
	[Country] [varchar](50) NULL,
 CONSTRAINT [PK_City] PRIMARY KEY CLUSTERED 
(
	[CityID] ASC
)) ON [PRIMARY]
GO

CREATE TABLE [dbo].[Person](
	[PersonID] [int] IDENTITY(1,1) NOT NULL,
	[Name] [varchar](50) NULL,
	[Address] [varchar](50) NULL,
	[CityID] [int] NULL,
 CONSTRAINT [PK_Person] PRIMARY KEY CLUSTERED 
(
	[PersonID] ASC
)) ON [PRIMARY]
GO

ALTER TABLE [dbo].[Person]  WITH CHECK ADD  CONSTRAINT [FK_Person_City] FOREIGN KEY([CityID])
REFERENCES [dbo].[City] ([CityID])
GO

ALTER TABLE [dbo].[Person] CHECK CONSTRAINT [FK_Person_City]
GO

CREATE PROCEDURE [dbo].[spGetAllPersons]
AS

SELECT PersonID, Person.Name AS PersonName, Address, City.Name AS CityName, Country
FROM Person, City
WHERE Person.CityID = City.CityID
FOR XML AUTO, ELEMENTS
GO

```

**Comment by CorsairRO** [comment]
Hello,

I needed as well the Xml Type there as I have lots of sp that uses it and the generated classes stays unbuildable. I;ve analyzed the .ttincludes provided by Simon in the generator.

Quick and dirty hack that works: In File: EF.Reverse.POCO.Core.ttinclude

Method: public static readonly Func<DataColumn, string> WriteStoredProcReturnColumn = col => {
in my version Is at line 3993.

Last parameter in return string.Format ( .... 

Replace columnName with: columnName.StartsWith("XML_") ? "XmlValue" : columnName


If you have many xml resultsets from there I don't know how to do it.


This is my next issue now to unify in generator the Resultsets coming from UNION queries that the tool unfortunately sees them as different resultsets even if there are not.

If the admins have another solution for that it is ok, but for now I can build my project.

**Comment by sjh37** [comment]
Released in v3.3.0
